src/molecules/lists/IconList/__tests__/icon_list.spec.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+//import sinon from 'sinon';
```

Comment:
take this out - i'm thinking we should probably remove it from the plop template at some point bc we never really use it/ use jest's mocking functionality instead